pkg/util/fileutil.go

```diff
@@ -0,0 +1,54 @@
+/*
+Package util provides utility functions for OSD servers and drivers.
+Copyright 2017 Portworx
```

Comment:
2017? You had these files just sitting around for two years ;-)

Reply:
fixed in 846b50e